lms/templates/discussion/_filter_dropdown.html

```diff
@@ -1,71 +1,44 @@
+               class="field-label">
 
-<%def name="render_category(categories, category, topic_list)">
-    <li class="forum-nav-browse-menu-item"
-        id='${category | n, decode.utf8}'
-        role="option"
-    >
-        <span class="forum-nav-browse-title">${category}</span>
-        <ul class="forum-nav-browse-submenu" role="presentation">
-            <% topic_list.append(category) %>
-            ${HTML(render_dropdown(categories[category], topic_list))}
-            <% topic_list.remove(category) %>
-        </ul>
-    </li>
-</%def>
+            <span class="field-label-text">{% trans "Filter Topics" as tmsg%} {{tmsg|force_escape}}</span>
+
+            <input type="text"
+                   id="forum-nav-browse-filter-input"
+                   role="combobox"
+                   aria-expanded="true"
+                   aria-owns="discussion_topics_listbox"
+                   aria-autocomplete="list"
+                   class="forum-nav-browse-filter-input"
+                   placeholder="{% trans 'filter topics' as tmsg%} {{tmsg|force_escape}}">
 
-<div class="forum-nav-browse-menu-wrapper" style="display: none" aria-label="${_("Discussion topics list")}">
-    <form class="forum-nav-browse-filter" autocomplete="off">
-        <label for="forum-nav-browse-filter-input" aria-label='${_("Filter Topics")}' class="field-label">
-            <span class="field-label-text">${_("Filter Topics")}</span>
-            <input type="text" id="forum-nav-browse-filter-input" role="combobox" aria-expanded="true" aria-owns="discussion_topics_listbox" aria-autocomplete="list" class="forum-nav-browse-filter-input" placeholder="${_("filter topics")}">
             <span class="icon fa fa-filter" aria-hidden="true"></span>
         </label>
     </form>
-    <ul class="forum-nav-browse-menu" role="listbox" id="discussion_topics_listbox">
-        <li class="forum-nav-browse-menu-item forum-nav-browse-menu-all" role="option" id="all_discussions">
-            <span class="forum-nav-browse-title">${_("All Discussions")}</span>
+
+    <ul class="forum-nav-browse-menu"
+        role="listbox"
+        id="discussion_topics_listbox">
+
+        <li class="forum-nav-browse-menu-item forum-nav-browse-menu-all"
+            role="option"
+            id="all_discussions">
+            <span class="forum-nav-browse-title">{% trans "All Discussions" as tmsg%} {{tmsg|force_escape}}</span>
         </li>
-        <li class="forum-nav-browse-menu-item forum-nav-browse-menu-following" role="option" id="posts_following">
+
+        <li class="forum-nav-browse-menu-item forum-nav-browse-menu-following"
+            role="option"
+            id="posts_following">
             <span class="icon fa fa-star" aria-hidden="true"></span>
-            <span class="forum-nav-browse-title">${_("Posts I'm Following")}</span>
+            <span class="forum-nav-browse-title">{% trans "Posts I'm Following" as tmsg%} {{tmsg|force_escape}}</span>
```

Comment:
Formatting can be improved:
`tmsg%}` > `tmsg %}`

21 instances of as tmsg%} in this PR should be as tmsg %} (missing space before %})